edgelet/Cargo.lock

```diff
@@ -181,11 +181,12 @@ dependencies = [
 
 [[package]]
 name = "config"
-version = "0.8.0"
+version = "0.9.3"
```

Comment:
Why do you need these dep changes?

Reply:
I had to update it cause using an enum just for their strings such as PullPolicy wasn't working

Here's a tracking git hub issue-
https://github.com/mehcode/config-rs/issues/74